altair/vegalite/v5/api.py

```diff
@@ -1011,11 +1041,14 @@ def to_dict(
                 vegalite_spec["$schema"] = SCHEMA_URL
 
             # apply theme from theme registry
-            the_theme = themes.get()
-            # Use assert to tell type checkers that it is not None. Holds true
-            # as there is always a default theme set when importing Altair
-            assert the_theme is not None
-            vegalite_spec = utils.update_nested(the_theme(), vegalite_spec, copy=True)
+            if theme := themes.get():
+                vegalite_spec = utils.update_nested(theme(), vegalite_spec, copy=True)
+            else:
```

Comment:
The previous comment doesn't consider the two following scenarios:

1. Assertions may not be present at runtime https://docs.astral.sh/ruff/rules/assert/
2. How a user would be expected to resolve the issue of there being no active theme. I imagine this assertion failing wouldn't be helpful on its own